test/categorical_algebra/CSets.jl

```diff
@@ -243,4 +243,44 @@ colim = pushout(α, β)
 @test !is_natural(α′) # Vertex labels don't match.
 @test_throws ErrorException pushout(α′, β)
 
+# Functorial Data Migration
+@present TheoryDDS(FreeSchema) begin
```

Comment:
Do we have this theory in theories? Seems like something that we should have, maybe.

Reply:
You mean exported from Catlab somewhere? Right now it's only in the unit tests in various places but DDS is a classic example, so probably makes sense to make it official.